app/src/main/res/values/styles.xml

```diff
@@ -189,4 +189,10 @@
         <item name="android:background">@drawable/round_rectangle_ripple</item>
     </style>
 
+    <style name="HomeMenuButton">
```

Comment:
NIT: how about extending this from MainMenuButton and only change the size?

Reply:
OK. Updated!